wordpress-mobile/WordPress-Android
# Fixes issue preventing changes on a homepage from being previewed

**Fixes:** https://github.com/wordpress-mobile/gutenberg-mobile/issues/2832

## Description
Fixes issue preventing changes on a homepage from being previewed. This is achieved by appending the page slug to the preview url based on the observation [discussed here](https://github.com/wordpress-mobile/gutenberg-mobile/issues/2832#issuecomment-732999563)

## To test:

1. Go to My Sites > + > Create a new WordPress.com site and complete the site creation flow.
1. Go to My Site > Pages > Home and select any layout from the "Choose a design" modal.
1. Edit your home page content.
1. Tap "Preview".
1. Try editing different pages created by headstart and try previewing changes to those pages.
1. Verify that the preview has the latest changes

PR submission checklist:
